docs/IPC_MESSAGES.md

```diff
@@ -190,7 +190,28 @@ Response:
     }
 }
 ```
-
+### `UpdateNewContractOnDeployment` message
+Request:
+```
+{
+    id : <unique_request_id>,
+    type : UpdateNewContractOnDeployment,
+    address : ...,
+    bytecode : [Secret Contract Address]
+    delta : [First delta]
```

Comment:
```suggestion
    delta : {address, key, data}
```

Reply:
I'm debating with you on this: to add my suggestion in order to align it with other delta requests or to save space since we have the key & address information already and stay with your proposal, what do you say?